### ISSUE TITLE
[BUG] Analysis fails for Nexus Public app

### ISSUE THREAD

**Issue by pranavgaikwad** [konveyor/analyzer-lsp#421]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

latest

### Priority

Critical

### Current Behavior

Analysis fails with panic

### Expected Behavior

Analysis shouldn't fail 

### How Reproducible

Always (Default)

### Steps To Reproduce

1. Run analysis against https://github.com/sonatype/nexus-public
2. Use targets `openjdk`
3. Notice that analysis fails with a panic

### Environment

```markdown
- **Analyzer Image SHA** latest
- **JDTLS  base image SHA**
- **Provider Settings JSON**
```

**Comment by jmle** [comment]
Fixed in https://github.com/konveyor/analyzer-lsp/pull/425, I think this can be closed now

**Comment by shawn-hurley** [comment]
Closed by #425
